src/session.js

```diff
@@ -46,15 +47,20 @@ export default class Session extends EventEmitter {
         if (withPayload)
             payloadScript = isIFrame ? this._getIFramePayloadScript() : this._getPayloadScript();
 
-        return mustache.render(TASK_TEMPLATE, {
+        var taskScript = mustache.render(TASK_TEMPLATE, {
             cookie:               cookies.replace(/'/g, "\\'"),
             sessionId:            this.id,
+            isFirstPageLoad:      this.pageLoadCount === 0,
             serviceMsgUrl:        serverInfo.domain + '/messaging',
             ie9FileReaderShimUrl: serverInfo.domain + '/ie9-file-reader-shim',
             crossDomainPort:      serverInfo.crossDomainPort,
             payloadScript:        payloadScript,
             referer:              referer
         });
+
+        this.pageLoadCount++;
```

Comment:
This is the completely opposite to what I've suggested in previous comment: at the first increment `this.pageLoadCount`, then compare it with `1` not zero.